### ISSUE TITLE
CIP-0119 - Inconsistent field name

### ISSUE THREAD

**Issue by palas** [cardano-foundation/CIPs#945]
I've noticed when addressing [this issue](https://github.com/IntersectMBO/cardano-cli/issues/906) that while the schema calls [this field](https://github.com/cardano-foundation/CIPs/blob/d08e2671148a4f970ad9dab6ea82c9e94ddf32af/CIP-0119/cip-0119.common.schema.json#L76) `"type"`, in [this example](https://github.com/cardano-foundation/CIPs/blob/d08e2671148a4f970ad9dab6ea82c9e94ddf32af/CIP-0119/examples/drep.jsonld#L47) it is written as `"@type"`. We probably should make this consistent. And I imagine we should favour `"@type"` for consistency with [CIP-0100](https://github.com/cardano-foundation/CIPs/tree/d08e2671148a4f970ad9dab6ea82c9e94ddf32af/CIP-0100)

**Comment by rphair** [comment]
Thanks @palas - I can see the consistency.  I don't work with related tooling so I'm tagging some reviewers & stakeholders to choose which term is canonical & agree upon a plan to change the other one.  If there's no obvious choice about which is canonical, or uncertain implications in changing it, we'll put it up for discussion on Discord and/or the next CIP meeting.

cc @Thomas-Upfield @Ryun1 @gitmachtl @perturbing @Crypto2099 @Quantumplation

**Comment by gitmachtl** [comment]
@palas i agree. `@type` is one of the registered keywords for json-ld format.

**Comment by rphair** [comment]
@gitmachtl this is likely a "stupid question" but does that mean we _should_ or _shouldn't_ use `@type`?

**Comment by rphair** [comment]
@palas please confirm here if you'd like to submit a pull request standardising on `@type` in the schemas & we can likely put it through at or before the CIP meeting next week.

**Comment by palas** [comment]
Sorry, I just saw the message. I am happy to make a PR to address this and also https://github.com/cardano-foundation/CIPs/issues/949

**Comment by rphair** [comment]
yes (and I also made https://github.com/cardano-foundation/CIPs/issues/949#issuecomment-2545695497 before seeing this), we appreciate the help @palas & I'll review and pass them along ASAP as soon as you submit the pull requests.
